### ISSUE TITLE
Review of science for L1- diagnostics

### ISSUE THREAD

**Issue by seanpcoultas** [metoppv/improver#41]
As a producers of StaGE level 1 data I want to review the science derivations so that we can ensure the quality of the data we produce

Related issues: #I, #J

A decription of the problem, and information on what has been done and ideas for future work is available here: https://exxreldocs:8099/display/~fiona.rust/Generation+of+Level+1+data
This work will need splitting down into smaller tickets, which should be priorities and refined before starting work.

Acceptance criteria:
 * To be confirmed

**Comment by fionaRust** [comment]
Some ideas on some of the things we could do to cover this are available here:
https://exxreldocs:8099/display/~fiona.rust/Generation+of+Level+1+data

**Comment by fionaRust** [comment]
I am currently reviewing the diagnostic derivations and the tickets in the backlog to try and make sure that we are being consistent in our derivations, and to prepare for extra derivations required for new diagnostics

**Comment by fionaRust** [comment]
@bjwheltor  can this be removed from the IMPROVER backlog?

**Comment by bjwheltor** [comment]
Being covered as part of the science improvement work for StaGE, so can be closed as an IMPROVER issue.
